### PR TITLE
cleaned files and directories

### PR DESCRIPTION
**Pull request by naqvia**
As per our conversation I have cleaned up the tmp files, renamed scripts and folders, deleted un-used cod and scripts, etc. I chose IntGeVaP as the tools name (we can change later); so I named the main scripts that. 
 
Can this be reviewed quickly... so we could merge? 

cc @jharenza @jungkim2

### CODE REVIEW

File: IntGeVaP/IntGeVaP-C.R [hunk 1]
```diff
@@ -0,0 +1,420 @@
+################################################################################
+# 01-annotate_variants.R
```
Comment (jharenza) [inline IntGeVaP/IntGeVaP-C.R:2]: I think it is fine to retain the original titles of the scripts, specifying CAVATICA, user-input. I don't really like the name of the tool being the name of the R script. Something like:
```
01-annotate_variants_CAVATICA_input.R
01-annotate_variants_custom_input.R
```

Is this accurate to say?

File: IntGeVaP/IntGeVaP-C.R [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+
+## function for filtering
+##gnomAD, variant af and depth filtering 
+gnomad_filtering <- function(clinvar_vcf) {
+  clinvar_vcf <- clinvar_vcf %>% 
+                  mutate(variant_depth = if_else( as.integer( str_match(INFO, "DP\\=(\\d+)")[, 2])  > filter_variant_depth, "PASS","FAIL")) %>% 
+                  mutate(gnomad_af     = if_else( as.numeric( str_match(INFO, "gnomad_3_1_1_AF_non_cancer\\=(0\\.\\d+)\\;")[,2])  > filter_variant_af, "PASS","FAIL")) %>% 
+                  mutate(variant_af    = if_else(as.integer(str_match(Sample, ":(\\d+)\\,(\\d+)") [,3]) / ( (as.integer(str_match(Sample, ":(\\d+)\\,(\\d+)") [,2]) ) + as.integer(str_match(Sample, ":(\\d+)\\,(\\d+)") [,3] )) > filter_variant_af, "PASS", "FAIL"))
+  return(clinvar_vcf)
+}
```
Comment (jharenza) [inline IntGeVaP/IntGeVaP-C.R:111]: later, we should pull out any functions and place in a util folder, then source them

File: IntGeVaP/IntGeVaP-C.R [hunk 3]
```diff
@@ -0,0 +1,420 @@
...
+  }
+}
+
+## fix spelling and nomenclature inconsistencies
+results_tab_abridged <- results_tab_abridged %>% mutate(final_call = replace(final_call, final_call == "Likely benign", "Likely_benign"))
+results_tab_abridged <- results_tab_abridged %>% mutate(final_call = replace(final_call, final_call == "Uncertain significance", "Uncertain_significance"))
+results_tab_abridged <- results_tab_abridged %>% mutate(final_call = replace(final_call, final_call == "Benign PVS1", "Benign"))
+results_tab_abridged <- results_tab_abridged %>% mutate(final_call = replace(final_call, final_call == "Pathogenic PVS1","Pathogenic"))
+results_tab_abridged <- results_tab_abridged %>% mutate(final_call = replace(final_call, final_call == "Likely pathogenic","Likely_pathogenic")) %>% distinct()
+
```
Comment (jharenza) [inline IntGeVaP/IntGeVaP-C.R:416]: you should make this one command in the future to clean up

File: IntGeVaP/README.md [hunk 4]
```diff
@@ -42,12 +42,12 @@ autopvs1 file (input/*autopvs1.txt)
 
 ##### run #####
 ```
-bash download_db_files.sh 
-Rscript 02-annotate_variants_user.R --vcf input/testing_010423_VEP.vcf --multianno input/testing_010423.hg38_multianno.txt --intervar input/testing_010423_intervar.hg38_multianno.txt.intervar --autopvs1 input/testing_010423_autopvs1.txt --clinvar input/clinvar_20211225.vcf.gz --output SRRT0182 --submission input/variant_summary.txt --submission_summary input/submission_summary.txt 
+bash download_db_files.sh
+Rscript 02-annotate_variants_user.R --vcf input/testing_010423_VEP.vcf --multianno input/testing_010423.hg38_multianno.txt --intervar input/testing_010423_intervar.hg38_multianno.txt.intervar --autopvs1 input/testing_010423_autopvs1.txt --clinvar input/clinvar_20211225.vcf.gz --output SRRT0182 --submission input/variant_summary.txt --submission_summary input/submission_summary.txt
```
Comment (jharenza) [inline IntGeVaP/README.md:46]: this second script name needs to be updated

**Review by jharenza — CHANGES_REQUESTED** [review]
Need to update script documentation, and I think rename the R scripts

File: IntGeVaP/IntGeVaP-C.R [hunk 5]
```diff
@@ -0,0 +1,420 @@
...
+# 01-annotate_variants.R
+# written by Ammar Naqvi
+#
+# This script annotates variants based on clinVar, and recomputes score and call 
+# for intervar and autoPVS1
+#
+# usage: Rscript 01-annotate_variants.R --vcf <vcf file> 
+#                                       --intervar <intervar file> 
+#                                       --autopvs1 <autopvs1 file>
+#                                       --clinvar  'yyyymmdd'
```
Comment (jungkim2) [inline IntGeVaP/IntGeVaP-C.R:11]: For --clinvar we need to replace 'yyyymmdd' to clinvar_yyyymmdd.vcf.gz

File: IntGeVaP/README.md [hunk 6]
```diff
@@ -42,12 +42,12 @@ autopvs1 file (input/*autopvs1.txt)
...
 ```
 
 ## Workflow chart  
-![workflow](https://github.com/diskin-lab-chop/pathogenicity-assessment/blob/annotations/analyses/germline-pathogenecity_flow.png)
+![workflow](readme_figs/germline-pathogenecity_flow.png)
```
Comment (jungkim2) [inline IntGeVaP/README.md:50]: we can replace this once I draw the figure for the table

File: IntGeVaP/README.md [hunk 7]
```diff
@@ -42,12 +42,12 @@ autopvs1 file (input/*autopvs1.txt)
```
Comment (jungkim2) [inline IntGeVaP/README.md:42]: We need to add in variant_summary.txt.gz AND submission_summary.txt.gz 
Also need to mention, those two files need to be unzipped.

File: IntGeVaP/IntGeVaP-C.R [hunk 8]
```diff
@@ -0,0 +1,420 @@
...
+input_autopvs1_file <- file.path(input_dir,"7aefaa8b-3dbe-4fe2-afb1-6f95d132ec87.autopvs1.chr20.tsv")
+input_submission_file <- file.path(input_dir,"variant_summary.txt")
+input_summary_submission_file <- file.path(input_dir,"submission_summary.txt")
+
+sample_name <-  "test"
+gnomad_variable <- "Freq_gnomAD_genome_ALL"
+gnomad_af <- 0.001
+filter_variant_depth = 15
+## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ###
+
```
[the remark this answers is not in the document]
Comment (jharenza) [reply IntGeVaP/IntGeVaP-C.R:99]: I think we wanted this as a user option, while this is default, but not seeing it being created with options

File: IntGeVaP/IntGeVaP-C.R [hunk 9]
```diff
@@ -0,0 +1,420 @@
...
+  make_option(c("--intervar"), type = "character",
+              help = "input intervar file"),
+  make_option(c("--multianno"), type = "character",
+              help = "input multianno file"),
+  make_option(c("--autopvs1"), type = "character",
+              help = "input autopvs1 file"),
+  make_option(c("--clinvar"), type = "character",
+              help = "specific clinVar file (format: clinvar_20211225.vcf.gz)"), ## https://ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf_GRCh38/archive_2.0/2022clinvar_20211225.vcf.gz 
+  make_option(c("--submission"), type = "character",
+              help = "specific submission file (format: variant_summary_2023-02.txt.gz)"),
```
[the remark this answers is not in the document]
Comment (jharenza) [reply IntGeVaP/IntGeVaP-C.R:50]: This is odd - will test on our end, but a gz file is read in exactly the same way in R

File: IntGeVaP/IntGeVaP-C.R (hunk 2 above)
Comment (naqvia) [reply IntGeVaP/IntGeVaP-C.R]: agreed

**Comment by naqvia** [comment]
I think I've addressed all the issues above (minus the util folder idea and combining some of the code).

**Pull request merged**